# HG changeset patch
# User Ben Abbott <bpabbott@mac.com>
# Date 1290386264 18000
#      Sun Nov 21 19:37:44 2010 -0500
# Node ID f0478684bc2fa0257bc960323394085edbfbf78d
# Parent  107e7476a5da4955e944145c0b72111cda9dfa90
__go_draw_axes__.m: Fix plot titles so they are visible for gnuplot 4.4.x (31711).

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,10 +1,12 @@
-2010-11-21  Ben Abbott <bpabbot@mac.com>
-
+2010-11-21  Ben Abbott <bpabbott@mac.com>
+
+	* plot/__go_draw_axes__.m: Fix plot titles so they are visible for
+	gnuplot 4.4.x (31711).
 	* general/cell2mat.m: Return empty matrix for empty cell.
 
 2010-11-21  Kai Habel  <kai.habel@gmx.de>
 
 	* (plot/uigetfile.m, plot/uiputfile.m): Set default directory to pwd
 
 2010-11-21  Kai Habel  <kai.habel@gmx.de>
 
diff --git a/scripts/plot/__go_draw_axes__.m b/scripts/plot/__go_draw_axes__.m
--- a/scripts/plot/__go_draw_axes__.m
+++ b/scripts/plot/__go_draw_axes__.m
@@ -142,25 +142,19 @@ function __go_draw_axes__ (h, plot_strea
 
     if (! isempty (axis_obj.title))
       t = get (axis_obj.title);
       if (isempty (t.string))
         fputs (plot_stream, "unset title;\n");
       else
         [tt, f, s] = __maybe_munge_text__ (enhanced, t, "string");
         fontspec = create_fontspec (f, s, gnuplot_term);
-        fprintf (plot_stream, "set title \"%s\" %s %s",
+        fprintf (plot_stream, "set title \"%s\" %s %s;\n",
                  undo_string_escapes (tt), fontspec,
                  __do_enhanced_option__ (enhanced, t));
-        if (nd == 3
-            && __gnuplot_has_feature__ ("screen_coordinates_for_{lrtb}margin"))
-          fprintf (plot_stream, " offset screen 0, screen %.3f;\n", pos(4)/5);
-        else
-          fprintf (plot_stream, ";\n");
-        endif
       endif
     endif
 
     if (! isempty (axis_obj.xlabel))
       t = get (axis_obj.xlabel);
       angle = t.rotation;
       colorspec = get_text_colorspec (axis_obj.xcolor, mono);
       if (isempty (t.string))
